sql/telemetry_derived/clients_scalar_probe_counts_v1/query.sql

```diff
@@ -115,6 +121,25 @@ WITH flat_clients_scalar_aggregates AS (
     os = 'Windows' and channel = 'release' AS sampled,
   FROM clients_scalar_aggregates_v1),
 
+logged_min_max AS (
```

Comment:
nit:  `log_min_max`, logged doesn't seem to right when talking taking the log of something.